testing-and-tuning/sample_conf.json

```
@@ -9,11 +9,14 @@
     "address": "Street Address 1",
```

I see this probably isn't super clear because there are so many moving pieces for input and output (and different names where things could be more consistent), but this `translation_map` is used to define which column from a source CSV maps to one of our required columns, with the key being our column and the value being the column name from the source CSV. So in the line
```
household_street_address1 = translation_lookup(row, "address street", column_maps)
```
we look to this config, key "address street" to see what column we should be reading from the source CSV from. So the key here needs to match the lookup being done in the code. 
```suggestion
    "address street": "Street Address 1",
```